### PR TITLE
base docker file

### PR DESCRIPTION
**Pull request by zzzming**

### CODE REVIEW

File: docker/Dockerfile.base [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM python:3.11
```
Comment (nicoloboschi) [inline docker/Dockerfile.base:1]: what about using a fixed version ?

File: docker/Dockerfile.multistage [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# The runtime image, used to just run the code provided its virtual environment
+FROM python:3.11-slim-buster as runtime
+
+ENV VIRTUAL_ENV=/app/.venv \
+    PATH="/app/.venv/bin:$PATH"
+
+COPY --from=builder ${VIRTUAL_ENV} ${VIRTUAL_ENV}
+
+# run the app
```
Comment (nicoloboschi) [inline docker/Dockerfile.multistage:27]: please drop comments

**Review by nicoloboschi — CHANGES_REQUESTED** [review]
@zzzming I don't understand this pr. should this include preinstalled ragstack-ai package?

File: docker/Dockerfile.multistage (hunk 2 above)
Comment (zzzming) [reply docker/Dockerfile.multistage]: fixed

**Pull request merged**